Tabloid/client/src/modules/categoryManager.js

```diff
@@ -0,0 +1,7 @@
+const _apiUrl = "/api/category";
+
+
+export const getAllCategories = () => {
+    return fetch(_apiUrl)
```

Comment:
If you do require auth to do this action, you'll need to get a token first and pass it into the headers of this request.